packages/react/src/PageMenu/PageMenu.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { Login, Menu } from '@amsterdam/design-system-react-icons'
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { PageMenu } from './PageMenu'
+import '@testing-library/jest-dom'
+import { Icon } from '../Icon'
+
+describe('page menu', () => {
+  it('renders a page menu with children', () => {
+    const { container } = render(
+      <PageMenu>
+        <PageMenu.Link href="#">English</PageMenu.Link>
+        <PageMenu.Link href="#">
+          Inloggen Mijn Amsterdam <Icon svg={Login} size="level-7" />
```

Comment:
The icon interface changed, right? It should be `<PageMenu.Link href="#" icon={Login}>`